internal/mode/static/nginx/config/servers.go

```diff
@@ -487,6 +489,37 @@ func generateProxySetHeaders(filters *dataplane.HTTPFilters) []http.Header {
 	return append(proxySetHeaders, headers...)
 }
 
+func generateAddHeaderDirectives(filters *dataplane.HTTPFilters) []http.Header {
```

Comment:
Is there any relationship between response headers and `RequestURLRewrite`, similar to what is seen in `generateProxySetHeaders` ([code](https://github.com/nginxinc/nginx-gateway-fabric/blob/e521f06aed06d864f358fc20036375cdb7b4c2a3/internal/mode/static/nginx/config/servers.go#L454-L461))? This [link](https://gateway-api.sigs.k8s.io/reference/spec/#gateway.networking.k8s.io/v1.HTTPURLRewriteFilter) said that "HTTPURLRewriteFilter defines a filter that modifies a request during forwarding.". Does it mean that `HTTPURLRewriteFilter` only affects requests?